cvat/apps/engine/media_extractors.py

```diff
@@ -662,7 +662,12 @@ def save_as_chunk(self, images, chunk_path):
                 output = io.BytesIO()
                 if self._dimension == DimensionType.DIM_2D:
                     pil_image = rotate_within_exif(Image.open(image))
```

Comment:
I will suggest an optimization. Let's use the original file if it is without any rotations (output = content of the file).

Reply:
Initially it was done to fix the bug with exif rotated images in original chunks which used in automatic annotations. 
As discussed, there was different options and it was decided to rotate original chunks.

Reply:
But I agree, there may be optimization if rotate_within_exif did not rotate the file at all (no one if was triggered). In this case we may write original file.